tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -327,6 +327,36 @@ def testee(a: cases.IFloatField) -> gtx.Field[[IDim], int64]:
     )
 
 
+def test_astype_on_tuples(cartesian_case):
+    @gtx.field_operator
+    def cast_tuple(
+        a: cases.IFloatField, b: cases.IFloatField
+    ) -> tuple[gtx.Field[[IDim], int64], gtx.Field[[IDim], int64]]:
+        return astype((a, b), int64)
+
+    @gtx.field_operator
+    def combine(a: cases.IFloatField, b: cases.IFloatField) -> gtx.Field[[IDim], int64]:
+        packed_tuple = cast_tuple(a, b)
+        return packed_tuple[0] + packed_tuple[1]
```

Comment:
```suggestion
```
This seems an unnecessary distraction from what the test is supposed to cover. Calling `cast_tuple` directly should be fine. Returning a tuple will likely fail with the dace backend, but just mark the test with `@pytest.mark.uses_tuple_returns` until dace supports it. Additionally I would use `cases.verify` with values that are non-integral (e.g. `1.1`) such that if the cast doesn't happen, but `verify_with_default_data` just uses an integer, it doesn't silently work.